src/main/java/seedu/duke/AddCommand.java

```diff
@@ -12,6 +13,7 @@ public AddCommand(InternshipList internshipList) {
     public void execute(String[] args) {
         String role = "";
         String company = "";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/yy");
         YearMonth startDate = YearMonth.parse("01/00");
         YearMonth endDate = YearMonth.parse("01/00");
```

Comment:
I think the formatter should be passed in as an argument in these 2 lines as well right (so as to handle the default case where no -from and -to flags are provided by the user) ?